### ISSUE TITLE
printer.cfg replacement in the docs?

### ISSUE THREAD

**Issue by 1980alexGS** [VzBoT3D/VzBoT-Vz330#61]
Hello - can the file here on Github named 330AWD-printer.cfg replace the printer.cfg linked at https://docs.vzbot.org/vz330_mellow/electronics/Printer_Config - the link there seems to be broken? Thanks

**Comment by EduardoMDSousa** [comment]
Yes 330awd is the printer.cfg just rename it

**Comment by 1980alexGS** [comment]
Thanks Eduardo - link at https://docs.vzbot.org/assets%5Cimages%5Cmanual%5Cvz330_mellow%5Celectronics%5CPrinter_config%5Cprinter.cfg still broken

**Comment by EduardoMDSousa** [comment]
Yes i see , the page admin need to Change the file name, becase was Changed. 👍
